src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -79,10 +79,14 @@ def _authentication_context_factory(tenant, cache):
 
 
 def _load_tokens_from_file(file_path):
-    all_entries = []
     if os.path.isfile(file_path):
-        all_entries = get_file_json(file_path, throw_on_empty=False) or []
-    return all_entries
+        try:
+            return get_file_json(file_path, throw_on_empty=False) or []
+        except Exception as ex:
```

Comment:
The exception is too broad. Figure out the exact types of the exception will be thrown.

Reply:
Okay, I scoped down to 2 types which should capture most of actionable errors, I think. The exception types raised by libraries are not that well documented, so i don't know what would fall through the crack, but we can catch up per incoming user reports